tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -2123,4 +2081,51 @@ public static JSONObject getLevelMenuJson(User me) throws JSONException {
         }
         return levels;
     }
+
+    private static final boolean NORMALIZE_USER_TABLE_ORGS = true;
+
+    /**
+     * Make sure each Organization in the user table has the normalized form of its name.
+     * The normalized form is the one that matches the enum name.
+     * For example, Organization.oracle has names "oracle", "Oracle", "sun", and "Sun Micro",
+     * according to Organization.java; its normalized name is "oracle".
+     * Organization.longnow has names "The Long Now Foundation", "Long Now", "PanLex", and
+     * "Utilka Foundation"; its normalized name is "longnow".
+     */
+    private void normalizeUserTableOrgs() {
```

Comment:
this ensures that the org names in the db exactly match the enum names in Organization.java